src/main/java/org/fundacionjala/coding/manuel/movies/Customer.java

```diff
@@ -0,0 +1,83 @@
+package org.fundacionjala.coding.manuel.movies;
+
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+/**
+ * Created by Juan Manuel on 3/18/2018.
+ */
+public class Customer {
+    private String name;
+    private List<Rental> rentals;
+
+    /**
+     * Constructor.
+     * @param name .
+     */
+    public Customer(String name) {
+        this.name = name;
+        rentals = new ArrayList<>();
+    }
+
+    /**
+     * Method to add a rental.
+     * @param arg .
+     */
+    public void addRental(Rental arg) {
+        rentals.add(arg);
+    }
+
+    /**
+     * Method getter.
+     * @return name .
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * Method statement.
+     * @return result .
+     */
+    public String statement() {
+        String result = String.format("%s %s%s", "Rental Record for", getName(), "\n");
+        for (Rental each : rentals) {
+            //show figures for this rental
+            result = result.concat(String.format("%s%s%s", "\t", each.getMovie().getTitle(), "\t"));
```

Comment:
In this case you should use the StringBuilder because it is more efficient in loops.